src/test/java/com/jeeconf/hibernate/performancetuning/nplusone/entity/Account.java

```diff
@@ -5,9 +5,10 @@
 
 import javax.persistence.*;
 
-@Entity
 @Getter
 @Setter
+@Entity(name = "NPlusOneAccount")
```

Comment:
It is unclear why did you introduce entity names and don't use them anywhere.

Reply:
I suppose they made this rule more strict in 5.4.0.CR1. And now it is not possible to have different entities with same name even not using then in JPQL directly:

https://hibernate.atlassian.net/browse/HHH-13149
https://hibernate.atlassian.net/browse/HHH-13060
https://github.com/hibernate/hibernate-orm/commit/817e4637645ef725af831ef8260a8059ce3a1090
https://docs.jboss.org/hibernate/orm/5.4/userguide/html_single/Hibernate_User_Guide.html#entity-pojo-mapping

Reply:
I know that. The question was more about usage of aliases in existing queries instead of packages.

Reply:
Sorry, I don't quite understand.
Using aliases in queries it was existing notation, I followed it.

Reply:
You added aliases to all entities, so in existing queries you could replace package usage with these aliases.

Reply:
All queries already use aliases.

Reply:
Sorry, my bad. I assumed we are still rely on packages.